example/README.md

```diff
@@ -1,4 +1,85 @@
 # Example
+Here is a collection of OpenTelemtry-Go examples showcasing basic functionality.
+
+## OTLP
+This example demonstrates how to export trace and metric data from an
+application using OpenTelemetry's own wire protocal OTLP. We will also walk
+you through configuring a collector to accept OTLP exports.
+
+### How to run?
+
+#### Prequisites
+- go >=1.14 installed
+- `GOPATH` is configured
+- (optional) OpenTelemetry collector is available
+
+#### Start the Application
+An example application is included in `example/otlp`. It simulates the process
+of scribing a spell scroll (e.g. in [D&D](https://roll20.net/compendium/dnd5e/Spell%20Scroll#content)).
+The application has been instrumented and exports both trace and metric data
+via OTLP to any listening receiver. To run it:
+
+```sh
+go get -d go.opentelemetry.io/otel
+cd $GOPATH/go.opentelemetry.io/otel/example/otlp
+go run main.go
+```
+
+The application is currently configured to transmit exported data to
+`localhost:55680`. See [main.go](otlp/main.go) for full details.
+
+Note, if you don't have a receiver configured to take in metric data, the
+application will complain about being unable to connect.
+
+#### (optional) Configure the Collector
```

Comment:
Good point, fixed

Reply:
Also, does it make sense to move it before the application config? The collector needs to be configured first right? We can add a sentence like "We need to configure the collector first just so that when data are pushed from the application the collector is around to collect the telemetry data.".

Reply:
Ahh yep, that's a good point. Done